docker/nuke.sh

```diff
@@ -6,3 +6,7 @@ cd $DIR
 # Tear down and clean up all DataHub-related containers, volumes, and network
 docker-compose -p datahub down -v
 docker-compose rm -f -v
+
+# Tear down ingestion container
+cd ingestion
```

Comment:
maybe `(cd ingestion && docker-compose -p datahub down -v)` ? 

That way the script ends in the same directory?

Reply:
Done!